### ISSUE TITLE
1.1.1 Override NodeController with routing configuration won't work

### ISSUE THREAD

**Issue by itkg-mraiffe** [open-orchestra/open-orchestra-cms-bundle#1712]
When I want to override NodeController but overriding the route did not work:

```
open_orchestra_front_node:
    path: /node/{nodeId}
    defaults: { _controller: MyFrontBundle:Node:show }
    methods: [GET]
```
I have to override open_orchestra_front.router.document_to_value_object.class to do the job.

**Comment by alavieille** [comment]
@itkg-mraiffe can you add context for this issue.

**Comment by itkg-canne** [comment]
@alavieille
We need to override the node controller in our front-office in order to add an ACL system. Our proble is that the class used for the route is hardcoded inside two OO service : `OpenOrchestra\FrontBundle\Routing\DatabaseRouteLoader` and `OpenOrchestra\FrontBundle\Routing\Database\TransformerRouteDocumentToValueObjectTransformer`.
So, for now, we need to override these 2 services in order to make our controller used.

There is at least 2 solutions to enhance that :
 - get the _controller default route option from the Symfony routing system in order to use the one defines in routing configuration (something like `$router->getRouteCollection()->get('open_orchestra_front_node')->getDefault('_controller')`),
 - use a configuration parameter to define path of controller.

**Comment by itkg-nanne** [comment]
fixed in 1.2: https://github.com/open-orchestra/open-orchestra-front-bundle/pull/169
fixed in 1.1: https://github.com/open-orchestra/open-orchestra-front-bundle/pull/170
